pkg/html/releasehtml/html.go

```diff
@@ -214,7 +210,7 @@ func canaryTestFailures(all, prevAll []sippyprocessingv1.FailingTestResult) stri
 			<th>Test Name</th><th>Pass Rate</th>
 		</tr>
 	`
-	template := `
+	tmpl := `
```

Comment:
what's the motivation for this change?

Reply:
This was from the linter catching an import shadow. That package imports `template`, but you're also assigning a value to the symbol `template`. Later, if you try to access anything in the `template` module from within that scope, you won't be able to. This one bit me a couple of times over the years.

Reply:
ah hah, makes sense, thanks